extensions/azcli/package.json

```diff
@@ -26,101 +26,16 @@
-        "title": "%azdata.config.title%",
+        "title": "%az.config.title%",
         "properties": {
           "azcli.logDebugInfo": {
             "type": "boolean",
             "default": false,
-            "description": "%azdata.config.debug%"
-          },
-          "azcli.acceptEula": {
-            "type": "string",
-            "default": "prompt",
-            "enum": [
-              "dontPrompt",
-              "prompt"
-            ],
-            "enumDescriptions": [
-              "%azdata.acceptEula.dontPrompt.description%",
-              "%azdata.acceptEula.prompt.description%"
-            ],
-            "description": "%azdata.acceptEula.description%"
-          },
-          "azcli.install": {
-            "type": "string",
-            "default": "prompt",
-            "enum": [
-              "dontPrompt",
-              "prompt"
-            ],
-            "enumDescriptions": [
-              "%azdata.install.dontPrompt.description%",
-              "%azdata.install.prompt.description%"
-            ],
-            "description": "%azdata.install.description%"
-          },
-          "azcli.update": {
-            "type": "string",
-            "default": "prompt",
-            "enum": [
-              "dontPrompt",
-              "prompt"
-            ],
-            "enumDescriptions": [
-              "%azdata.update.dontPrompt.description%",
-              "%azdata.update.prompt.description%"
-            ],
-            "description": "%azdata.update.description%"
-          },
-          "azcli.requiredUpdate": {
-            "type": "string",
-            "default": "prompt",
-            "enum": [
-              "dontPrompt",
-              "prompt"
-            ],
-            "enumDescriptions": [
-              "%azdata.update.dontPrompt.description%",
-              "%azdata.update.prompt.description%"
-            ],
-            "description": "%azdata.requiredUpdate.description%"
+            "description": "%az.config.debug%"
           }
         }
       }
     ],
-    "commands": [
-      {
-        "command": "azcli.acceptEula",
-        "title": "%azdata.acceptEula.command.name%",
-        "category": "%command.category%"
-      },
-      {
-        "command": "azcli.install",
-        "title": "%azdata.install.command.name%",
-        "category": "%command.category%"
-      },
-      {
-        "command": "azcli.update",
-        "title": "%azdata.update.command.name%",
-        "category": "%command.category%"
-      }
-    ],
-    "menus": {
-      "commandPalette": [
-        {
-          "command": "azcli.acceptEula",
-          "when": "!azcli.eulaAccepted"
-        },
-        {
-          "command": "azcli.install",
-          "when": "!azcli.found"
-        },
-        {
-          "command": "azcli.update",
-          "when": "azcli.found"
-        }
-      ]
-    },
     "resourceDeploymentOptionsSources": [
       {
         "id": "arc.controller.config.profiles"
```

Comment:
This should be updated as well to match https://github.com/microsoft/azuredatastudio/pull/16415/files#diff-60b11e728b29c436a9cacee3c5f1090507365fb71baa8a259c683c67c0d45354R13